include/geode/geometry/quality.hpp

```diff
@@ -39,4 +41,8 @@ namespace geode
 
     [[nodiscard]] double opengeode_geometry_api
         tetrahedron_collapse_aspect_ratio( const Tetrahedron& tetra );
+
+    template < index_t dimension >
+    [[nodiscard]] double opengeode_geometry_api triangle_angle_based_quality(
```

Comment:
```suggestion
    [[nodiscard]] double triangle_angle_based_quality(
```
Do not export templated functions in the header